gnovm/cmd/gno/test.go

```diff
@@ -90,6 +91,13 @@ func (c *testCfg) RegisterFlags(fs *flag.FlagSet) {
 		0,
 		"max execution time",
 	)
+
+	fs.BoolVar(
+		&c.withNativeFallback,
+		"with-native-fallback",
+		false,
+		"use stdlibs/* if present, otherwise use native (only for testing)",
```

Comment:
```suggestion
		"use stdlibs/* if present, otherwise use supported native Go packages",
```

(since this is in `gno test`, we know implicitly this is for testing)

I also think we should specify that it's only the _supported_ packages. Otherwise a reader might think that it can refer to all packages in the go stdlib.